PyDSS/utils/utils.py

```diff
@@ -156,16 +156,16 @@ def iter_elements(element_class, element_func):
 
     Examples
     --------
-    >>> import opendssdirect as dss
+    import opendssdirect as dss
```

Comment:
Why did you make these changes?

Reply:
Oh, I was getting an error while running it the way it was. It was trying to run those lines of code. Can revert to what it was if needed

Reply:
Was it failing in your IDE or in the terminal?  This section is in the docstring, so it shouldn't get executed.

Reply:
I tried it on the terminal. It doesn't fail there.

Reply:
It still fails for you, right?  That means there must be a problem.  Can you share a screenshot over chat?